wmn-data.json

```diff
@@ -130,7 +130,18 @@
         "m_code" : 200,
         "known" : ["mike", "greg"],
         "cat" : "coding",
-        "valid" : true
+        "valid" : true
       },

```

Comment:
Seeing a Ctrl-M here again. Please remove.
![image](https://user-images.githubusercontent.com/2359093/189784894-83ffc439-c4f3-41c2-8337-40a98640b6a3.png)

Reply:
@WebBreacher:
- Ctrl-M `removed`
- gitignore additions `removed`

Reply:
The Ctrl-M persists. I'll accept the PR and then manually change it.